src/include/planner/operator/ddl/logical_alter.h

```diff
@@ -6,17 +6,41 @@
 namespace kuzu {
 namespace planner {
 
+struct LogicalAlterPrintInfo final : OPPrintInfo {
+    common::AlterType alterType;
+    std::string tableName;
+    binder::BoundExtraAlterInfo* info;
```

Comment:
If this object points to the info stored in the logical operator, then the life cycle should be fine.

Otherwise, it's better to copy the object here for safety.